### DISCUSSION TITLE
Multiline input

### DISCUSSION THREAD

**Discussion by Mart-Bogdan** [evcxr/evcxr#346] [Q&A]
How do I enter multiple lines of code before executing?

I've been using Rust Rover from jet brains and in it evcxr always work in multi-line mode. You need to hit Ctrl+Enter to submit.

And also in Konsole when I paste from clipboard it is multi-line. But when I am just typing, each press of Enter key causes execution, unless you have un-closed bracket.

**Comment by davidlattimore** [comment]
In the REPL at the moment, having an open brace or something like that is the only way.

Using the jupyter kernel is one option, since it lets you enter multiple lines.

It'd be nice if the REPL had a way too, however last I checked, I don't think there was a way to do it with rustyline, which the repl uses for reading input.

**Comment by farazshaikh** [comment]
This is a bit of bummer :(, the only reason I use irust for now
This interpreter is great !
